src/remediations/fifi.integration.js

```diff
@@ -114,13 +114,15 @@ describe('FiFi', function () {
 
         test.skip('get connection status with false smartManagement but not configured with config manager', async () => {
             base.getSandbox().stub(config, 'isMarketplace').value(true);
-            base.getSandbox().stub(configManager, 'getCurrentState').resolves({
-                account: '654321',
-                state: {
-                    remediations: 'disabled'
-                },
+            base.getSandbox().stub(configManager, 'getCurrentProfile').resolves({
+                account_id: '654321',
+                active: true,
```

Comment:
Just FYI, this is the description for the `active` field in the schema in the API docs: `Boolean, Remote host configuration enabled state`
(I looked it up because I wasn't sure if we'd need to check this as well as the `remediations` field at some point )